pkg/webhooks/server.go

```diff
@@ -36,97 +34,63 @@ import (
 // MutationWebhook gets policies from policyController and takes control of the cluster with kubeclient.
 type WebhookServer struct {
 	server        http.Server
-	client        *client.Client
-	kyvernoClient *kyvernoclient.Clientset
+	Client        *client.Client
```

Comment:
Wats the reason behind Capitalizing the First character for each field?
With this, all variables are Exposed outside the struct. Only the ones needed to be exposed should be exposed, rest should be only available within package.

Reply:
They fields significantly simplify the implementation if they are exported - there are no apparent drawbacks to exporting it.

Reply:
If we expose all the variables, then we don't have the concept of private variables and public variables. All variables are public, and all external packages have access to it. `Defensive Programming`.

This will simplify the code for the developer, but at the cost of making everything in the package public.

Reply:
Ok, whats the problem with making it public - our repo is not designed to be used as a library anyway, so this should not be a concern

Reply:
OK!
The suggestion was a good design point of view.
Again, The capitalizing of variables makes the element expose outside the package. In other languages like Java and C++, with keywords `public` and `private` we control the visibility. And these fields are important as they define the behavior of struct/class.
Even though the pkg is not used in a library, we shouldn't disregard the good coding principles.

I have no further comments to add to the above, as its a design choice.

Reply:
There can be exceptions for best practices. For example its widely regarded that global variables should be avoided as much as possible - however there are scenarios where the benefits of using globals outweigh its cons i.e cache, db connections .etc. You will find globals inside std lib of golang as well in log and flags.

This is a similar scenario where there are no real drawbacks to exporting fields - while the benefits are apparent.

Reply:
@shivdudhani requested changes have been made kindly review